builtInFunctions/multiESDTNFTTransfer_test.go

```diff
@@ -828,3 +829,13 @@ func TestESDTNFTMultiTransfer_NotEnoughGas(t *testing.T) {
 	_, err = transferFunc.ProcessBuiltinFunction(sender.(vmcommon.UserAccountHandler), sender.(vmcommon.UserAccountHandler), vmInput)
 	assert.Equal(t, err, ErrNotEnoughGas)
 }
+
+func TestComputeInsufficientQuantityESDTError(t *testing.T) {
+	t.Parallel()
+
+	resErr := computeInsufficientQuantityESDTError([]byte("my-token"), 0)
+	require.Equal(t, errors.New("insufficient quantity for token: my-token").Error(), resErr.Error())
+
+	resErr = computeInsufficientQuantityESDTError([]byte("my-token-2"), 5)
+	require.Equal(t, errors.New("insufficient quantity for token: my-token-2 nonce 5").Error(), resErr.Error())
```

Comment:
:+1: these tests should be written as you did because of backwards incompatibility issues that might appear.